### DISCUSSION TITLE
apparently this could make up to 2.5x faster the SD/Dreambooth inference

### DISCUSSION THREAD

**Discussion by elBlacksmith** [TheLastBen/fast-stable-diffusion#594] [Ideas]
Just for your information, yesterday it was announced on the Strange Diffusion subReddit, and it's already published.
I don't understand the details, but maybe it's useful: https://github.com/VoltaML/voltaML

**Comment by TheLastBen** [comment]
I saw it, I'll test it soon

**Comment by newacc88000000** [comment]
up

**Comment by TheLastBen** [comment]
Not yet on point, we'll wait for updates from the Volta team

**Comment by Pasha49** [comment]
with SD:
https://github.com/VoltaML/voltaML-fast-stable-diffusion

only optional on colab (images differ)
![image](https://user-images.githubusercontent.com/17119725/203996001-38bf60a7-d6bd-4a8d-bfb7-30466af3a66e.png)
![image](https://user-images.githubusercontent.com/17119725/203996033-3f3a4e4d-43ce-4e33-a899-20cff5d3db33.png)
